server/src/index.ts

```diff
@@ -31,6 +32,9 @@ const io = new Server(socketServer, {
   },
 });
 
+// Begin searching and collecting Garbage (old messages)
+scheduleCron();
```

Comment:
can you put this at the bottom of this page, right before all the commented out stuff.